components/chip/chip.vue

```diff
@@ -1,7 +1,6 @@
 <template>
-  <span class="d-chip">
+  <span :class="'d-chip'">
```

Comment:
no reason to do this if it's just a string, leave it as an unbound attribute: `class="d-chip"`